utils/spacewalk-common-channels.ini

```diff
@@ -3020,3 +3020,165 @@ gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni
+
+[raspbian-9-armhf-firmware-uyuni]
+label    = raspbian-9-armhf-firmware-uyuni
+name     = Raspbian 9 (stretch) armhf Firmware for Uyuni
+archs    = arm-deb
+repo_type = deb
+checksum = sha256
+base_channels = raspbian-9-pool-armhf-uyuni
+repo_url = https://archive.raspbian.org/raspbian/dists/stretch/firmware/binary-armhf/
+
+[raspbian-9-armhf-non-free-uyuni]
+label    = raspbian-9-armhf-non-free-uyuni
+name     = Raspbian 9 (stretch) armhf Non-Free for Uyuni
+archs    = arm-deb
+repo_type = deb
+checksum = sha256
+base_channels = raspbian-9-pool-armhf-uyuni
+repo_url = https://archive.raspbian.org/raspbian/dists/stretch/non-free/binary-armhf/
+
+[raspbian-9-armhf-rpi-uyuni]
+label    = raspbian-9-armhf-rpi-uyuni
+name     = Raspbian 9 (stretch) armhf RPi for Uyuni
+archs    = arm-deb
+repo_type = deb
+checksum = sha256
+base_channels = raspbian-9-pool-armhf-uyuni
+repo_url = https://archive.raspbian.org/raspbian/dists/stretch/rpi/binary-armhf/
+
+[raspbian-10-pool-armhf-uyuni]
+label    = raspbian-10-pool-armhf-uyuni
+checksum = sha256
+archs    = arm-deb
+repo_type = deb
+name     = Raspbian 10 (buster) pool for armhf for Uyuni
+gpgkey_url =
+gpgkey_id =
+gpgkey_fingerprint =
+repo_url = https://archive.raspbian.org/raspbian/dists/buster/main/binary-armhf/
+
+[raspbian-10-armhf-contrib-uyuni]
+label    = raspbian-10-armhf-contrib-uyuni
+name     = Raspbian 10 (buster) armhf Contrib for Uyuni
+archs    = arm-deb
+repo_type = deb
+checksum = sha256
+base_channels = raspbian-10-pool-armhf-uyuni
+repo_url = https://archive.raspbian.org/raspbian/dists/buster/contrib/binary-armhf/
+
+[raspbian-10-armhf-firmware-uyuni]
+label    = raspbian-10-armhf-firmware-uyuni
+name     = Raspbian 10 (buster) armhf Firmware for Uyuni
+archs    = arm-deb
+repo_type = deb
+checksum = sha256
+base_channels = raspbian-10-pool-armhf-uyuni
+repo_url = https://archive.raspbian.org/raspbian/dists/buster/firmware/binary-armhf/
+
+[raspbian-10-armhf-non-free-uyuni]
+label    = raspbian-10-armhf-non-free-uyuni
+name     = Raspbian 10 (buster) armhf Non-Free for Uyuni
+archs    = arm-deb
+repo_type = deb
+checksum = sha256
+base_channels = raspbian-10-pool-armhf-uyuni
+repo_url = https://archive.raspbian.org/raspbian/dists/buster/non-free/binary-armhf/
+
+[raspbian-10-armhf-rpi-uyuni]
+label    = raspbian-10-armhf-rpi-uyuni
+name     = Raspbian 10 (buster) armhf RPi for Uyuni
+archs    = arm-deb
+repo_type = deb
+checksum = sha256
+base_channels = raspbian-10-pool-armhf-uyuni
+repo_url = https://archive.raspbian.org/raspbian/dists/buster/rpi/binary-armhf/
+
+[fedora33]
+archs    = x86_64, s390x, aarch64, ppc64le
+checksum = sha256
+name     = Fedora 33 (%(arch)s)
+gpgkey_url = https://getfedora.org/static/fedora.gpg
+gpgkey_id = 9570FF31
+gpgkey_fingerprint = 963A 2BEB 0200 9608 FE67  EA42 49FD 7749 9570 FF31
+repo_url = https://mirrors.fedoraproject.org/metalink?repo=fedora-33&arch=%(arch)s
+dist_map_release = 33
+
+[fedora33-modular]
```

Comment:
Last time I checked, Fedora was not working with Uyuni at all because it provided a newer salt version, incompatible with our master. Do I understand we plan to use Fedora to test the bundle?

If so, this LGTM. But keep in mind we still need to add client tools repositories.

Reply:
It's for Salt Bundle only. Pau asked to add Fedora 33/34 for Salt Bundle, most probably someone asked for it, I don't remember the details.

Reply:
Salt Bundle for Fedora-33 see: https://build.opensuse.org/project/show/systemsmanagement:saltstack:bundle:Fedora-33

Reply:
@Shirocco88 , it's a build project of Salt Bundle, there are a list of packages which shouldn't be published for the channels synchronized to Uyuni as these packages are build dependencies only and cant break the dependencies of the OS.

Reply:
@vzhestkov :
thanks for your answer;
I want to evaluate Salt Bundle, for example with Debian-10;
what is the recommended way ?
i.) Download venv-salt-minion.deb 
or ii.) is there a possibility to mirror a repository with venv-salt-minion, build repo, bootstrap

any help welcome.

Reply:
@Shirocco88 `venv-salt-minion` package is only usable package from Salt Bundle project repos. So you need to sync only this package (in simple way just to sync all the repo and delete everything except venv-salt-minion). `venv-salt-minion` package is the only package required for bootstrapping, but please note that only Uyuni Master contains all the states aligned for `venv-salt-minion`. `/usr/share/susemanager/mgr_bootstrap_data.py` should be updated to use `venv-salt-minion` for creating bootstrap repo.

Reply:
Are Fedora 36 and future builds going to be supported by chance? Sadly, the previous admin used Fedora for some of the server builds and it would be nice to manage these alongside our other Linux distros.

Reply:
I suppose, no. It requires significant effort for building the Salt Bundle for it, but almost no requests to add these distros.

Reply:
Would I be able to add the repos to the spacewalk-common-channels.ini and use the saltstack installer instead? I was reading that Fedora needed to use the venv-minion instead so maybe that wouldn't be doable then?

Reply:
you can try to add the repos manually there and most probably these repos will be synched with no any issues, but the problem is salt here. We can't guaranee that Uyuni Server will work with salt-minion from upstream. Maybe better to try building venv-salt-minion for Fedora 36 by branching these project in OBS: https://build.opensuse.org/project/show/systemsmanagement:saltstack:bundle:Fedora35

Reply:
Honestly, I wouldn't know where to begin trying to make a build. If it was a config/yaml file and say build this with these packages I could do it but if it's doing programming like Python then I would have difficulties. Do you have links I could try to learn from?